server/main.lua

```diff
@@ -31,27 +31,11 @@ end, {help = "Unjail people from jail", params = {{name = "id", help = "target i
 -- send to jail and register in database
 RegisterServerEvent('esx_jail:sendToJail')
 AddEventHandler('esx_jail:sendToJail', function(target, jailTime)
-	local identifier = GetPlayerIdentifiers(target)[1]
+	local xPlayer = ESX.GetPlayerFromId(source)
 
-	MySQL.Async.fetchAll('SELECT * FROM jail WHERE identifier = @identifier', {
-		['@identifier'] = identifier
-	}, function(result)
-		if result[1] then
-			MySQL.Async.execute('UPDATE jail SET jail_time = @jail_time WHERE identifier = @identifier', {
-				['@identifier'] = identifier,
-				['@jail_time'] = jailTime
-			})
-		else
-			MySQL.Async.execute('INSERT INTO jail (identifier, jail_time) VALUES (@identifier, @jail_time)', {
-				['@identifier'] = identifier,
-				['@jail_time'] = jailTime
-			})
-		end
-	end)
-	
-	TriggerClientEvent('chat:addMessage', -1, { args = { _U('judge'), _U('jailed_msg', GetPlayerName(target), ESX.Math.Round(jailTime / 60)) }, color = { 147, 196, 109 } })
-	TriggerClientEvent('esx_policejob:unrestrain', target)
-	TriggerClientEvent('esx_jail:jail', target, jailTime)
+	if xPlayer.job ~= nil and xPlayer.job.name == 'police' then
```

Comment:
The job is never set to `nil`, but you can check for `xPlayer` itself

Reply:
Also instead of checking for the police job, I would like to see a config file for authorized jobs